uws/UWS/models.py

```diff
@@ -87,7 +117,7 @@ def add_job(self, id=None, href=None, phase=None, job=None):
 
 
 class JobRef(BaseUWSModel):
-    def __init__(self, id=None, phase=None, reference=None, xml_node=None, xml_namespace=None):
+    def __init__(self, id=None, phase=None, reference=None, xml_node=None, xml_namespace=None, uws_qnames=None):
```

Comment:
ah I see... qualified_names... maybe we can call this somehow that it is easier to understand and does not refer to the XML name. How about `uws_version` or `uws_flavour`?